README.md

```diff
@@ -133,20 +133,19 @@ git clone git@github.com:tawakuliKH/portofolio.git
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/217450945-12146905-c393-4be5-b406-04738cc28471.png)

- [ ] You are not using the correct linter setup because your linters setup should have (**Lighthouse, Webhint, Stylint, ESLint, node_modules checker**) :blush: but your linters setup is missing ESLint which finds errors in javascript files. Those errors can break your application functionality :fearful: So you should have ESLint in your linter setup :+1:

Note: You can add the proper linter setup from this [link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) :+1: